test/StoragelessTest/Http/SessionMiddlewareTest.php

```diff
@@ -371,8 +372,8 @@ public function testSessionTokenParsingIsDelayedWhenSessionIsNotBeingUsed()
     {
         /* @var $signer Signer|\PHPUnit_Framework_MockObject_MockObject */
         $signer = $this->createMock(Signer::class);
-
```

Comment:
> Could you clarify why this test was incorrect without the signature?

Description of #63 

> `SessionMiddleware#appendToken()` always calls `SessionInterface#isEmpty()` which [will initialise the session data](https://github.com/psr7-sessions/storageless/blob/609f52ede289f5dc9775a7df83104ca434a60fce/src/Storageless/Session/LazySession.php#L111).
> 
> The [test](https://github.com/psr7-sessions/storageless/blob/609f52ede289f5dc9775a7df83104ca434a60fce/test/StoragelessTest/Http/SessionMiddlewareTest.php#L370) only passes because the token is not signed and then [`Token#verify()`](https://github.com/lcobucci/jwt/blob/3.2.1/src/Token.php#L194) throws an exception which is silently ignored.

Commit message:

> Signature verification was never being delayed because an unsecured
token was being used and `Token#verify()` was throwing an exception
instead.

Do you want me to reword the commit message or is this enough?